rpc/releaselist.go

```diff
@@ -0,0 +1,18 @@
+package rpc
+
+import "capnproto.org/go/capnp/v3"
+
+type releaseList []capnp.ReleaseFunc
+
+func (rl releaseList) Release() {
+	for _, r := range rl {
+		r()
+	}
+	for i := range rl {
+		rl[i] = func() {}
```

Comment:
Nit:  I'm not sure what the compiler/runtime implications are of assigning nop functions, but I suspect it might allocate a new anonymous function pointer each time.  It's not a huge deal, but since we're already tweaking `releaseList`, maybe we should nil these out and add a nil check in the previous loop instead?

Reply:
I think it's more readable the way you suggest, so done (and I also merged the two loops) -- but functions can only be compared to nil, so  I'm like 99.9% sure this won't allocate; it should be able to just assign the same function pointer each time.